README.md

```diff
@@ -86,6 +86,16 @@ Installing (Docker)
 
 Follow [these instructions](https://github.com/ankicommunity/anki-devops-services#about-this-docker-image).
 
+Configuration: .env.local (Environment variables)
+* The environment variables can be found here: config/.env.example. 
+* The file also includes other development variables, but the notable ones are the ones with the prefix ANKISYNCD_
+* Environment variables will override the config files values (which is why I recommend you use them)
+* This is what we use in the Docker images (see: https://github.com/ankicommunity/anki-devops-services/blob/develop/services/anki-sync-server/examples/docker-compose.yml).
```

Comment:
My ton is quite conversational across this entire section. Could you update it to be more formal, if possible?